File: Tasks/Python Work/quadrant_finder.py
number_one = int(input("Enter the first number: "))
number_two = int(input("Enter the second number: "))

if (number_one > 0 and number_two > 0): 
	print("Quadrant 1")
elif (number_one < 0 and number_two > 0): 
	print("Quadrant 2")
elif (number_one < 0 and number_two < 0): 
	print("Quadrant 3")
elif (number_one > 0 and number_two < 0): 
	print("Quadrant 4")
elif (number_one == 0 and number_two == 0): 
	print("Origin")
elif (number_one != 0 and number_two == 0): 
	print("X-axis")
elif (number_one == 0 and number_two != 0): 
	print("Y-axis")

File: Tasks/Python Work/Java Work/QuadrantFinder.java
import java.util.Scanner;
	public class QuadrantFinder {
		public static void main(String[] args){
		Scanner input = new Scanner(System.in);

	System.out.println("Enter the first number");
	int numberOne = input.nextInt();

	System.out.println("Enter the second number");
	int numberTwo = input.nextInt();

if (numberOne > 0 && numberTwo > 0){
	System.out.println("Q1");
} else if (numberOne < 0 && numberTwo > 0){
	System.out.println("Q2");
} else if (numberOne < 0 && numberTwo < 0){
	System.out.println("Q3");
} else if (numberOne > 0 && numberTwo < 0){
	System.out.println("Q4");
} else if (numberOne == 0 && numberTwo == 0){
	System.out.println("Origin");
} else if (numberOne != 0 && numberTwo == 0){
	System.out.println("X-axis");
} else {
	System.out.println("Y-axis");
}
}
}


